pkg/server/exec_arm.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2020 Elotl Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"errors"
+	"github.com/elotl/itzo/pkg/api"
+	"github.com/elotl/wsstream"
+	"os/exec"
+)
+
+const (
+	wsTTYControlChan = 4
+)
+
+func (s *Server) runExec(ws *wsstream.WSReadWriter, params api.ExecParams) {
+	writeWSErrorExitcode(ws, "not supported on arm")
+	return
+}
+
+func (s *Server) runExecCmd(ws *wsstream.WSReadWriter, cmd *exec.Cmd, interactive bool) error {
+	return errors.New("not supported on arm")
+}
+
+func (s *Server) runExecTTY(ws *wsstream.WSReadWriter, cmd *exec.Cmd, interactive bool) error {
+	return errors.New("not supported on arm")
+}
+
```

Comment:
Is there anything that's x86-specific in exec.go?

Reply:
Downstream dependency: https://github.com/jandre/procfs/issues/11 Do you think this is anything we can easily fix there / in our fork?

Reply:
I guess possible solution could be switching to something similar with better support, e.g. https://github.com/prometheus/procfs

Reply:
actually it seems that will be almost a transparent change in terms of API

Reply:
Yeah, that's a lot better than stubbing out the whole file for ARM.